fontc_crater/src/ci.rs

```diff
@@ -242,44 +242,64 @@ fn make_targets(cache_dir: &Path, repos: &[FontSource]) -> ResolvedTargets {
             Ok(config) => config,
             Err(e) => {
                 result.failures.insert(repo.repo_url.clone(), e.to_string());
+                log::warn!(
+                    "failed to load repo '{}': '{e}'",
+                    repo.repo_path(cache_dir).display()
+                );
                 continue;
             }
         };
         let config = match Config::load(&config_path) {
             Ok(x) => x,
             Err(e) => {
                 result.failures.insert(repo.repo_url.clone(), e.to_string());
+                log::warn!("failed to load config {}", config_path.display());
                 continue;
             }
         };
-        let relative_config_path = config_path
-            .strip_prefix(cache_dir)
-            .expect("config always in cache dir");
-        let sources_dir = config_path
-            .parent()
-            .expect("config path always in sources dir");
-        // config is always in sources, sources is always in org/repo
-        let repo_dir = relative_config_path.parent().unwrap().parent().unwrap();
+        // we map repos to URLS in a special sources.json file
```

Comment:
I don't think I understand what "special" is trying to tell me? I wonder if we could describe what is in sources.json (or point to the description) more directly? - it's not entirely clear to me from this what I'm going to find there